test/videojs-contrib-hls.test.js

```diff
@@ -1466,6 +1468,18 @@ QUnit.test('the source handler supports HLS mime types', function(assert) {
   });
 });
 
+QUnit.test('source handler does not support sources when IE 10 or below', function(assert) {
+  videojs.browser.IE_VERSION = 10;
+
+  ['html5', 'flash'].forEach(function(techName) {
+    assert.ok(!HlsSourceHandler(techName).canHandleSource({
+      type: 'aPplicatiOn/x-MPegUrl'
```

Comment:
I know it shouldn't matter, but since this test is specific to IE 10 testing, may want to keep this is proper casing